images/ci-prune-buildcache/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM fedora:40
```

Comment:
Is there a reason to not just use the standard `python` base image we usually use for python-based cron jobs?

Reply:
Just standardization. If there are benefits to using Fedora that the `python` image doesn't have, I don't have any objections